user_sync/connector/umapi.py

```diff
@@ -183,7 +183,8 @@ def send_commands(self, commands, callback=None):
         if len(commands) > 0:
             action_manager = self.get_action_manager()
             action = action_manager.create_action(commands)
-            action_manager.add_action(action, callback)
+            if action:
```

Comment:
if there is a flag value returned on error, check for it explicitly.  Otherwise you might get surprised if someone defines `__bool__` (py2) or `__nonzero__` (py3) or `__len__` (both) on the class.
```suggestion
            if action is not None:
```

Reply:
Agreed - we should be explicit with None checks whenever possible.